SPCTR-2974 Fix: Call To Action - Resolved an issue of undefined `$combined_selectors` on frontend. (#3240)
brainstormforce/ultimate-addons-for-gutenberg: readme.txt

== Changelog ==

= x.x.x =

> * Fix: Call To Action - Resolved an issue of undefined variable on frontend.

Reviewer: ```suggestion
* Fix: Call To Action - Removed the undefined variable warning on frontend.
```

Author: Done
